firewall/interception/introspection.go

```diff
@@ -0,0 +1,78 @@
+package interception
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/safing/portbase/log"
+)
+
+var (
+	packetMetricsDestination string
+	metrics                  = &packetMetrics{
+		done: make(chan struct{}),
+	}
+)
+
+func init() {
+	flag.StringVar(&packetMetricsDestination, "packet-metrics", "", "Write packet metrics")
```

Comment:
Would suggest naming the flag `write-packet-metrics` and changing the description to `Write packet metrics to the specified file`. This makes it clear that these are written to a file.